notification-server/src/bin/service.rs

```diff
@@ -65,12 +67,25 @@ struct Args {
         default_value_t = 180  // 3 minutes
     )]
     google_client_max_interval_time_secs: u64,
+    #[clap(
+        long = "log-level",
+        default_value = "info",
+        help = "The maximum log level. Possible values are: `trace`, `debug`, `info`, `warn`, and \
```

Comment:
If you use the `log-level` option with for example `trace`, then every process/library logs everything. You could specify specific targets that are most relevant for this application (where errors could originate from), we often choose the following targets:

```
  let log_filter = tracing_subscriber::filter::Targets::new()
            .with_target(module_path!(), app.log_level)
            .with_target("tower_http", app.log_level);
            .with_target("tokio_postgres", app.log_level);

        tracing_subscriber::registry()
            .with(tracing_subscriber::fmt::layer())
            .with(log_filter)
            .init();
```


Additional targets (when you using the indexer as Soeren suggested)
            .with_target("ccd_indexer", app.log_level)
            .with_target("ccd_event_processor", app.log_level)